src/main/app/controller/PostController.ts

```diff
@@ -63,6 +69,14 @@ export class PostController<T extends AnyObject> {
       res.redirect(nextUrl);
     });
   }
+
+  private async save(req: AppRequest<T>, formData: Partial<Case>, eventName: string): Promise<CaseWithId | false> {
+    try {
+      return await req.locals.api.triggerEvent(req.session.userCase.id, formData, eventName);
+    } catch {
+      return false;
```

Comment:
Need logging

Reply:
`triggerEvent` will [catch any exceptions, log them, and rethrow](https://github.com/hmcts/nfdiv-frontend/blob/master/src/main/app/case/CaseApi.ts#L50) so we shouldn't need to do any logging here.